components/Pages/Payruns/ApproveDeclineModal/index.js

```diff
@@ -68,9 +68,13 @@ const ApproveDeclineModal = ({ openedModal, setOpenedModal, update, payRunId })
 
   const handleReject = async (notes) => {
     try {
-      await rejectPayRun(payRunId, notes);
-      pushNotification(`Invoice status changed`, 'success');
-      update();
+      if (request) {
+        await request(notes);
+      } else {
+        await rejectPayRun(payRunId, notes);
+        pushNotification(`Pay run status changed`, 'success');
```

Comment:
if you're using this for invoices as well then you need to change pay run to a prop

Reply:
added rejectRequest function